std_msgs/Header header
float64[] x
float64[] y
float64 exploration_time
int32 direction